plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -80,192 +197,724 @@ function executeCLI($scriptName, &$args) {
 
 		AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON);
 
+		[$filename, $username, $basePath] = $this->parseCommandLineArguments($scriptName, $args);
+
+		$this->validateUser($username);
+		$file = $this->createAndValidateCSVFile($filename);
+
+		$csvForInvalidRowsName = "{$basePath}/invalid_rows.csv";
+		$invalidRowsFile = $this->createAndValidateCSVFileInvalidRows($csvForInvalidRowsName);
+
+		// Imports
+		import('lib.pkp.classes.submission.SubmissionFile'); // constants.
+		import('lib.pkp.classes.file.FileManager');
+		import('lib.pkp.classes.core.Core');
+		import('classes.file.PublicFileManager');
+
+		// Cache variables
+		$processedRows = 0;
+		$failedRows = 0;
+		$this->_expectedRowSize = count($this->_expectedHeaders);
+
+		foreach ($file as $index => $fields) {
+			if (!$index) {
+				continue; // Skip headers
+			}
+
+			if (empty(array_filter($fields))) {
+				continue; // End of file
+			}
+
+			$processedRows++;
+
+			if (count($fields) < $this->_expectedRowSize) {
+				$reason = __('plugins.importexport.csv.csvDoesntContainAllFields');
+				$failedRows = $this->processFailedRow($invalidRowsFile, $fields, $reason, $failedRows);
+
+				continue;
+			}
+
+			$data = (object) array_combine($this->_expectedHeaders, array_pad(array_map('trim', $fields), $this->_expectedRowSize, null));
+			if (!$this->requiredFieldsPresent($data)) {
+				$reason = __('plugins.importexport.csv.requiredFieldsMissing');
+				$failedRows = $this->processFailedRow($invalidRowsFile, $fields, $reason, $failedRows);
+
+				continue;
+			}
+
+			// Format is:
+			// Press Path, Author string, title, abstract, series path, year, is_edited_volume, locale, URL to Asset, doi (optional), keywords list, subjects list, book cover image path, book cover image alt text, categories list
```

Comment:
I think this can be removed, we have this description on the `$_expectedHeaders`